services/github.spec.js

```diff
@@ -1,12 +1,14 @@
+/* eslint-env mocha */
+/* eslint func-names: "off" */
 const chai = require('chai');
 const chaiHttp = require('chai-http');
 const nock = require('nock');
 const chaiAsPromised = require('chai-as-promised');
 
 const config = require('../config');
-const github = require('../services/github');
+const github = require('./github');
 
-const should = chai.should();
+const should = chai.should(); // eslint-disable-line no-unused-vars
```

Comment:
No need to assign it to a variable and then disable a lint rule. Just invoke the function.